examples/multi_script_CLI.py

```diff
@@ -457,7 +465,7 @@ def menu_comment():
             2.Use username database
             """
             )
-            if stripped_input() == "1":
+            if "1" in sys.stdin.readline():
```

Comment:
why you changed it everywhere? Why the previous version worse?

Reply:
Yup that didn't work. Lot of users were complaining